src/app/sections/ProbeDetails.svelte

```diff
@@ -122,26 +170,53 @@ h2 {
     {/if}
     <div class=drawer-section>
         {#if $store.probe.description}
-            <h2 class=detail__heading--01>description</h2>
-            <div class='probe__description helper-text--01'>
+            <h2 class="detail__heading--01">description</h2>
+            <div class="probe-description helper-text--01">
                 {@html $store.probe.description}
             </div>
         {/if}
     </div>
     {#if $store.probe.bugs && $store.probe.bugs.length}
     <div class="drawer-section">
-            <h2 class=detail__heading--01>associated bugs</h2>
-            <div class="bug-list helper-text--01">
-            {#each $store.probe.bugs as bugID, i (bugID)}
-                <a
-                href='https://bugzilla.mozilla.org/show_bug.cgi?id={bugID}'
-                target="_blank">{bugID}</a>
-            {/each}
-            </div>
+        <h2 class="detail__heading--01">associated bugs</h2>
+        <div class="bug-list helper-text--01">
+        {#each $store.probe.bugs as bugID, i (bugID)}
+            <a
+            href='https://bugzilla.mozilla.org/show_bug.cgi?id={bugID}'
+            target="_blank">{bugID}</a>
+        {/each}
+        </div>
     </div>
     {/if}
-    <div class='drawer-section drawer-section--end'>
-        <h2 class=detail__heading--01>export</h2>
+    <div class="drawer-section">
```

Comment:
I would consider combining "extras" with the "available in" sections, with a goal to make the data labeling in the sections speak enough for itself. I left some additional comments below this one on how we might let the data speak for itself.